content/applications/hr/time_off.rst

```diff
@@ -169,96 +207,128 @@ employee works (hour, day, week, etc), they earn or *accrue* a specified amount
-:guilabel:`After this accrual's period`. By default, the first level begins once the time off is
-approved if the time off is based on an accrual plan. If :guilabel:`Immediately` is selected, then
-the next level begins according to the time frame set on the level. If :guilabel:`After this
-accrual's period` is selected, the next level does not begin until the first level is completed
-according to the rules set on it.
+Enter the following information on the form:
+
+- :guilabel:`Name`: enter the accrual plan name.
+- :guilabel:`Accrued Gain Time`: select when the employee begins to accrue time off, either
+  :guilabel:`At the start of the accrual period` or :guilabel:`At the end of the accrual period`.
+- :guilabel:`Carry-Over Time`: select when the employee received previously earned time. The options
+  are:
+
+  - :guilabel:`At the start of the year`: select this if the accrual rolls over on January 1 of the
+    upcoming year.
+  - :guilabel:`At the allocation date`: select this if the accrual rolls over as soon as time is
+    allocated to the employee.
+  - :guilabel:`Other`: select this option if neither of the other two options are applicable. When
+    selected, a :guilabel:`Carry-Over Date` field appears. Select the date using the two drop-down
+    menus, one for the day and one for the month.
+
+- :guilabel:`Based on worked time`: enable this option if time off accrual is determined by the
+  employee's worked hours. Days **not** considered as worked time do **not** contribute to the
+  accrual plan in Odoo.
+- :guilabel:`Milestone Transition`: this selection determines when employees move up to a new
+  milestone. If they qualify to change milestones in the middle of a pay period, select if the
+  employee changes milestones :guilabel:`Immediately` or :guilabel:`After this accrual's period`
+  (after the current pay period).
+
+  .. important::
+     The :guilabel:`Milestone Transition` field is **only** visible after a minimum of two
+     :ref:`rules <time_off/rules>` have been configured on the accrual plan.
+
+- :guilabel:`Company`: using the drop-down menu, select the company the accrual plan applies to. If
+  left blank, the accrual plan can be used for all companies. This field only appears in a
+  multi-company database.
+
+.. image:: time_off/accrual-plan-form.png
+   :align: center
+   :alt: An accrual plan form with all the entries filled out.
+
+.. _time_off/rules:
 
 Rules
 *****
 
-Rules must be created in order for the accrual plan to accrue time off.
+Rules must be created in order for employees to accrue time off from the accrual plan.
 
-To create a new rule, click the :guilabel:`Add A New Level` button right beneath the word `Rules`,
-and a :guilabel:`Create Level` pop-up form appears.
+To create a new rule, click the :guilabel:`New Milestone` button in the gray :guilabel:`Rules`
+section, and a :guilabel:`Create Milestone` pop-up form appears.
```

Comment:
This is a _modal_ UI element rather than a _pop-up_ :)
```suggestion
section, and a :guilabel:`Create Milestone` modal appears.
```

Reply:
Wow- I have never even heard that word before!